Products/ZenUI3/browser/resources/js/zenoss/support.js

```diff
@@ -0,0 +1,189 @@
+    singleton: true,
+    data: {
+        layout: 'border',
+        items: [{
+            itemId: 'supportbundles',
+            region: 'center',
+            xtype: 'basegridpanel',
+            uid: '/zport/dmd/support',
+            store: Ext.create('Zenoss.DirectStore', {
+                itemId: 'supportbundlestore',
+                root: 'bundles',
+                autoLoad: true,
+                totalProperty: 'totalCount',
+                pageSize: Zenoss.settings.supportBundlesGridBufferSize,
+                initialSortColumn: 'modDate',
+                initialSortDirection: 'DESC',
+                //totalProperty: 'totalCount',
+                model: 'Zenoss.model.SupportBundle',
+                directFn: Zenoss.remote.SupportRouter.getBundlesInfo,
+                listeners: {
+                    'load': function(store) {
+                        store.loaded = true;
+                    }
+                }
+            }),
+            columns: [{
+                id: 'fileName',
+                dataIndex: 'fileName',
+                header: _t('Filename'),
+                flex: 6,
+                sortable: true
+            },{
+                id: 'sizeFormatted',
+                dataIndex: 'sizeFormatted',
+                header: _t('Size'),
+                flex: 1,
+                sortable: true,
+            },{
+                id: 'modDate',
+                dataIndex: 'modDate',
+                header: _t('Date Modified'),
+                flex: 1,
+                sortable: true,
+                renderer: renderDate
+            }],
+            multiSelect: true,
+            title: _t('Support Bundles'),
+            tbar: {
+                itemId: 'support-toolbar',
+                items: [{
+                    itemId: 'gather-bundle',
+                    iconCls: 'add',
+                    text: _t('Add'),
+                    tooltip: _t('Gather a support bundle in the background'),
+                    handler: function() {
+                        var dialog = Ext.create('Zenoss.MessageDialog', {
+                            message: 'The support bundle will continue to be gathered even if you close this window.  The newly gathered support bundle will not appear until the process has completed.  The job log may be monitored for progress.',
+                            okHandler: function() {
+                                Zenoss.remote.SupportRouter.createSupportBundle({}, function(response) {
+                                    if (response.success) {
+                                        var win = Ext.create('Zenoss.JobLog', {
+                                            job: response.new_jobs,
+                                            title: _t('Create Support Bundle')
+                                        });
+                                        win.on('destroy', function() {
+                                            window.location.reload();
```

Comment:
maybe just refresh your store (or the whole panel if needed) instead of refreshing the entire page.